src/server_manager/web_app/ui_components/app-root.js

```diff
@@ -764,12 +768,8 @@ export class AppRoot extends mixinBehaviors
         });
   }
 
-  _computeIsDigitalOceanAccountConnected(digitalOceanAccountName) {
-    return Boolean(digitalOceanAccountName);
-  }
-
-  _computeHasManualServers(serverList) {
-    return this.serverList.filter(server => !server.isManaged).length > 0;
+  _hasManualServers(serverList) {
```

Comment:
Hmm. Does Polymer reevaluate this when `serverList` changes? How does it know to do that?

Reply:
This line tells Polymer to call the function when the serverList changes:
```html
<div class="servers-section" hidden\$="[[!_hasManualServers(serverList)]]">
```

Reply:
See https://polymer-library.polymer-project.org/3.0/docs/devguide/data-binding#annotated-computed